### PR TITLE
Ve 2173 - variant support in data-cslp

### PR DESCRIPTION
**Pull request by KANE-99**

### CODE REVIEW

File: src/live-preview.ts [hunk 1]
```diff
@@ -297,19 +305,25 @@ export default class LivePreview {
...
-                cslpTag.split(".");
-
+            const {
+                content_type_uid,
+                entry_uid,
+                locale,
+                variant,
+                fieldPath: field,
+            } = extractDetailsFromCslp(cslpTag);
+            console.log("🚀 ~ LivePreview ~ scrollHandler ~ variant:", variant);
```
Comment (alt-test) [inline src/live-preview.ts:315]: Let's remove this log

File: src/live-preview.ts [hunk 2]
```diff
@@ -321,7 +335,13 @@ export default class LivePreview {
                         content_type_uid,
                         locale,
                         entry_uid,
-                        field.join(".")
+                        variant,
+                        field
+                    );
+                    console.log(
```
Comment (alt-test) [inline src/live-preview.ts:341]: Remove the logs

**Review by alt-test — APPROVED** [review]
looks good!

File: src/live-preview.ts (hunk 1 above)
Comment (Deepak-Kharah) [inline src/live-preview.ts:315]: let's remove the logs

File: src/live-preview.ts (hunk 2 above)
Comment (Deepak-Kharah) [inline src/live-preview.ts:341]: Let's remove the logs

**Review by Deepak-Kharah — CHANGES_REQUESTED** [review]
Let's remove the logs

**Review by Deepak-Kharah — APPROVED** [review]
Looks good!

File: src/__test__/live-preview.test.ts [hunk 3]
```diff
@@ -18,6 +20,10 @@ describe("cslp tooltip", () => {
         titlePara.setAttribute("data-cslp", TITLE_CSLP_TAG);
         titlePara.setAttribute("data-test-id", "title-para");
 
+        const variantTitlePara = document.createElement("h3");
+        variantTitlePara.setAttribute("data-cslp", VARIANT_TITLE_CSLP_TAG);
+        variantTitlePara.setAttribute("data-test-id", "title-variant-para");
```
Comment (KANE-99) [inline src/__test__/live-preview.test.ts:25]: change it to `title-variant-heading`

**Pull request closed without merging**